common/src/main/res/values/strings.xml

```diff
@@ -1313,4 +1313,6 @@
     <string name="sensor_description_steps">Total steps taken since midnight from Health Connect</string>
     <string name="basic_sensor_name_blood_glucose">Blood glucose</string>
     <string name="sensor_description_blood_glucose">Last recorded blood glucose reading in milligrams per deciliter from Health Connect</string>
+    <string name="general">General</string>
+    <string name="sensor_worker">Sensor Worker</string>
```

Comment:
Maybe now is the time to rename this to something that makes more sense to users like "Sensor updates"?

Reply:
I am glad you said something :)  think that sounds like a good expectation